src/System.Json/pkg/System.Json.pkgproj

```diff
@@ -0,0 +1,34 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup>
+    <!-- Skipping SupportCheck for this package since we need to update the FrameworkList for this to work-->
+    <SkipSupportCheck>true</SkipSupportCheck>
```

Comment:
You should be able to remove this now and add an entry for $(AllXamarinFrameworks).  The buildtools update with this included went in yesterday.